### PR TITLE
[NON MODULAR] Vape carts + THC tweaks

### PR DESCRIPTION
**Pull request by linnpap**
## About The Pull Request
Adds a variety of vape carts, which are now the only way to refill vapes. Also removes lipolicide from cannabis, and adds increased hunger as an effect of THC.

## Why It's Good For The Game
If I Do Not Have Dr Breens Blue Kush Reserve In Skyrat I Am Going To FUcking Kill Myself

## Changelog
:cl:
add: Added vape carts
del: Removed lipolicide from cannabis
add: Adds increased hunger as an effect of THC
/:cl:

### CODE REVIEW

**Comment by FlamingLily** [comment]
Good change, but I do have to qualms. First, feature freeze, this should really wait until the codebase isn't dying

Two: https://github.com/tgstation/tgstation/pull/59360 you've been beaten too the punch on the weed tweaks

**Comment by linnpap** [comment]
> Two: [tgstation/tgstation#59360](https://github.com/tgstation/tgstation/pull/59360) you've been beaten too the punch on the weed tweaks

yeah i know, its just that its a separate reagent thats getting added

**Comment by Sonic121x** [comment]
I don't like the idea about haveing CUM inside a vape cart
atleast make it sell from kinkmate instead?

**Comment by StarHorder** [comment]
I've seen Duboris do crazy antag things with their vape, would this still allow those things to be done?

File: modular_skyrat/modules/morenarcotics/items/vapecarts.dm [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	list_reagents = list()
+	reagent_flags = OPENCONTAINER
+	var/labelled = FALSE
+
+/obj/item/reagent_containers/vapecart/empty/attack_self(mob/user)
+	if(reagents.total_volume > 0)
+		to_chat(user, "<span class='notice'>You empty [src] of all reagents.</span>")
+		reagents.clear_reagents()
+
+/obj/item/reagent_containers/vapecart/empty/attackby(obj/item/I, mob/user, params)
```
Comment (tf-4) [inline modular_skyrat/modules/morenarcotics/items/vapecarts.dm:40]: No single letter variables

File: modular_skyrat/modules/morenarcotics/items/vapecarts.dm [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+/obj/item/reagent_containers/vapecart/reddiesel
+	name = "Resistance Red Diesel cart"
+	desc = "Seems to be endorsed by a real scientist!"
+	list_reagents = list(/datum/reagent/drug/thc = 20, /datum/reagent/consumable/dr_gibb = 10, /datum/reagent/toxin = 5)
+	custom_price = PAYCHECK_MEDIUM
+
+/obj/item/reagent_containers/vapecart/pwrgame
+	name = "Pwr Haze cart"
+	desc = "When did Pwr Game get into the cart business?"
+	list_reagents = list(/datum/reagent/drug/thc = 20, /datum/reagent/consumable/pwr_game = 10, /datum/reagent/cum = 5 , /datum/reagent/toxin = 5) //pwr game literally is cum lets get real
```
Comment (tf-4) [inline modular_skyrat/modules/morenarcotics/items/vapecarts.dm]: Let's not even go there.

```suggestion
	list_reagents = list(/datum/reagent/drug/thc = 20, /datum/reagent/consumable/pwr_game = 10, /datum/reagent/drug/cocaine = 5 , /datum/reagent/toxin = 5) //pwr game literally is crack lets get real
```

**Review by tf-4 — CHANGES_REQUESTED** [review]

**Comment by YakumoChen** [comment]
Don't like this one at all..
Toxin in a smokable doesnt make sense. Taking toxins from drugs is generally limited to raw plants (like ambrosia) or hard drugs. Save taking toxins for addiction or overdose effects please. 

It also does a lot of damage quickly so no sane person would smoke the carts since they're fairly lethal.

On top of that, vaping makes clouds which means you multiply the reagents from smoking AND you're poisoning everyone around you with just a vendor available cart.

Cum in something smokable is a huge NO please dont even. You're going to immediately pref break everyone around you. Someone is going to buy one not knowing what's in it and be pref broke immediately. This is such a hugely bad idea.

**Comment by Funce** [comment]
Feature Freeze

**Comment by FlamingLily** [comment]
> Cum in something smokable is a huge NO please dont even. You're going to immediately pref break everyone around you. Someone is going to buy one not knowing what's in it and be pref broke immediately. This is such a hugely bad idea.

I didn't even see the cum cart, that's such a bad idea, holy crap. Like, I'm a _supporter_ of mechancial erp and goddamn, just, no...

**Comment by linnpap** [comment]
> 
> 
> I've seen Duboris do crazy antag things with their vape, would this still allow those things to be done?

you can get empty vape carts that you can fill with reagents, so yeah

**Comment by Gandalf2k15** [comment]
Reopen after FF

**Pull request closed without merging**